platforms/quorum/configuration/roles/setup/geth/Readme.md

```diff
@@ -16,40 +16,31 @@ This task check if geth is already installed or not
 ##### Output Variables
     geth_stat_result: This variable stores the info on availibility on geth binary
 
-#### 2. Check quorum repo dir exists
-This task check if geth is already installed or not
+#### 2. Create geth directory
```

Comment:
Incorrect. `stat` module checks for the presence of file/directory, it doesn't create them.